src/App.tsx

```diff
@@ -20,12 +25,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(e) => {
+                  setQuery(e.target.value);
+                }}
```

Comment:
Move this to handleChange

```suggestion
                onChange={handleChange}
```